### ISSUE TITLE
DeprecationWarning: pkg_resources is deprecated as an API.

### ISSUE THREAD

**Issue by c02y** [ranger/ranger#2868]
#### Runtime Environment

- Operating system and version: `Arch Linux`
- Terminal emulator and version: `Kitty 0.28.1`
- Python version: `3.11.3`
- Ranger version/commit: `ranger-master commit:1a007158`
- Locale: all are `en_US.UTF-8`

#### Current Behavior
When starting ranger, error message is displayed:
```
/usr/bin/ranger:4: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  __import__('pkg_resources').run_script('ranger-fm==1.9.3', 'ranger')
```

Ranger seems working fine except displaying the error message before being started.
#### Steps to reproduce
1. 
2. 
3. 

#### Traceback

```

```

**Comment by toonn** [comment]
I'm not sure where the warning's coming from. We don't have `pkg_resources` anywhere in the codebase. Maybe we're somehow using its API through something like `imp`?

**Comment by c02y** [comment]
Yeah, maybe, or maybe setuptools related, I'm not sure the root cause

FYI:
```
>> cat /usr/bin/ranger
#!/usr/bin/python -O
# EASY-INSTALL-SCRIPT: 'ranger-fm==1.9.3','ranger'
__requires__ = 'ranger-fm==1.9.3'
__import__('pkg_resources').run_script('ranger-fm==1.9.3', 'ranger')
```

**Comment by vejkse** [comment]
There are relevant issues for setuptools and pip:
- https://github.com/pypa/setuptools/issues/3981
- https://github.com/pypa/pip/issues/11975

I don’t really understand what is going on, but this doesn’t look like a ranger issue.

**Comment by c02y** [comment]
Problem is already solved, I reinstall ranger(ranger-git 1.9.3.678.g1a007158-1), the old ranger was installed from old git source.
